client/js/widgets/widget.js

```diff
@@ -1347,22 +1347,27 @@ export class Widget extends StateManaged {
       }
 
       if(a.func == 'SORT') {
-        setDefaults(a, { key: 'value', reverse: false, collection: 'DEFAULT' });
+        setDefaults(a, { key: 'value', reverse: false, collection: 'DEFAULT', rearrange: true });
         let collection;
-        let reverse = a.reverse ? 'in reverse' : '';
+        let reverse = (a.reverse && !Array.isArray(a.reverse)) ? ' in reverse' : '';
+        let key = asArray(a.key).map((k)=>{
+          if(k !== null && typeof k.key == 'string')
+            return `'${k.key}'`;
+          return typeof k == 'string' ? `'${k}` : k;
```

Comment:
I might miss something but it feels like there's a `'` missing somewhere.

Reply:
Yep. Just JSON editor routine debugging output, but still should be fixed.